lib/documents.js

```diff
@@ -3338,6 +3338,280 @@ function copyConstraints(suggestConstraints, searchConstraints) {
   return destination;
 }
 
+Documents.prototype.transformAll = function transformAllDocuments(options) {
+  return transformAllDocumentsImpl.call(
+      this, options
+  );
+};
+
+function transformAllDocumentsImpl(jobOptions){
+  let path = '/v1/internal/forestinfo';
+  let connectionParams = this.client.getConnectionParams();
+  let requestOptions = mlutil.copyProperties(connectionParams);
+
+  requestOptions.method = 'GET';
+  requestOptions.headers = {
+    'Accept':       'application/json',
+  };
+
+  requestOptions.path = mlutil.databaseParam(connectionParams, path);
+  let operation = new Operation(
+      'read forestInfo', this.client, requestOptions, 'empty', 'single'
+  );
```

Comment:
Could any of the setup for getting forest information be refactored into a common function to reduce duplication of code and thus make maintenance easier?

Reply:
Created a new issue to refactor the code - https://github.com/marklogic/node-client-api/issues/652